integrations/launchdarkly/main.py

```diff
@@ -0,0 +1,92 @@
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from client import LaunchDarklyClient, ObjectKind
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from typing import Any
+
+
+def initialize_client() -> LaunchDarklyClient:
+    return LaunchDarklyClient(
+        launchdarkly_url=ocean.integration_config["launchdarkly_host"],
+        api_token=ocean.integration_config["launchdarkly_token"],
+    )
+
+
+@ocean.on_resync(kind=ObjectKind.AUDITLOG)
+async def on_resync_auditlog(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    logger.info(f"Listing Lauchdarkly resource: {kind}")
+    launchdarkly_client = initialize_client()
+    async for auditlogs in launchdarkly_client.get_paginated_resource(kind, limit=20):
+        logger.info(f"Received {kind} batch with {len(auditlogs)} items")
```

Comment:
you can remove this log from each of the resync methods and instead just add it to the `get_paginated_resource` function that way all kinds will have this log automatically when they use the `get_paginated_resource` function